src/backend/joanie/core/utils/contract_definition.py

```diff
@@ -61,9 +89,29 @@ def generate_document_context(contract_definition, user, order=None):
                 )
                 if order
                 else _("<COURSE_NAME>")
-            )
+            ),
+            "code": (order.course.code if order else _("<COURSE_CODE>")),
```

Comment:
For readability, we could define variables
```
course_code = _("<COURSE_CODE>")
course_name = …
if order:
    course_code = order.course.code
    course_name = …
```
then just use `course_code` and others in the returned dict
```
"code": course_code,
```
wdyt ?